Prevent `aggregate_last_24_hours` option from affecting global state (#16033)
DataDog/integrations-core: snowflake/CHANGELOG.md

## Unreleased

***Fixed***:

> * Fixed bug where setting the `aggregate_last_24_hours` option to `true` would not be honored when other instances had it set it to `false` (the default) ([#16033](https://github.com/DataDog/integrations-core/pull/16033))

Reviewer: ```suggestion
* Fixed bug where setting the `aggregate_last_24_hours` option to `true` was not honored when other instances had it set to `false` (the default) ([#16033](https://github.com/DataDog/integrations-core/pull/16033))
```

Author: Thanks!
